threading/channels.nim

```diff
@@ -270,11 +270,13 @@ proc `=copy`*[T](dest: var Chan[T], src: Chan[T]) =
   dest.d = src.d
 
 proc trySend*[T](c: Chan[T], src: sink Isolated[T]): bool {.inline.} =
-  ## Tries to send a message to a channel.
+  ## Tries to send an item `src` to a channel `c`.
   ##
-  ## The memory `src` is moved, not copied. Doesn't block.
+  ## The memory of `src` is moved, not copied. 
+  ## Returns immediately once the lock to the channel was acquired and an 
+  ## attempt to send it was made.
```

Comment:
```suggestion
  ## Doesn't block waiting for the available space in the channel and returns
  ## immediately after an attempt to send an item was made.
  ## 
  ## .. warning:: Blocking is still possible if another thread uses the blocking
  ##    version of `send`/`recv` and waits for the data/space to appear in the
  ##    channel, thus holding the internal lock to channel's buffer.
```

Reply:
Applied this change (did a small rewording by splitting up the first sentence into two and adding some doc-links to the warning block).